piflash/pyFlashGui/.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "python.pythonPath": "/usr/bin/python"
+    "python.pythonPath": "D:\\Python37\\python.exe"
```

Comment:
Is this specific for windows? This is not tge path on my machine.

Reply:
Yes, this is machine-specific. This python utility is for the manufacturer though, you won't need to run it.